src/index.html

```diff
@@ -2,13 +2,60 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyoLogo"
+          class="header__image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="link link--one">
+            <a href="#" class="item is-active">Apple</a>
+          </li>
+          <li class="link link--two">
```

Comment:
Why do you need extra classes as "link--one", "link--two" etc?

Reply:
I used them for easier navigation through the same items. I removed them because I do understand that they, in fact, don't have any practical use.